english/staypuft/carrot-nose/find-carrot-nose.md

```diff
@@ -1,7 +1,7 @@
 Your snowman has been built, but he can't breathe! He needs a nose, STAT! 
 You look around desperately, pushing aside the thick marshmallows, 
 straining to see any flashes of orange. You can't find a carrot and you're running out of time!
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.